src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2995,33 +3342,46 @@ public partial class CompareInfo
         public virtual int Compare(string string1, string string2) { return default(int); }
         public virtual int Compare(string string1, string string2, System.Globalization.CompareOptions options) { return default(int); }
         public override bool Equals(object value) { return default(bool); }
+        public static System.Globalization.CompareInfo GetCompareInfo(int culture) { return default(System.Globalization.CompareInfo); }
         public static System.Globalization.CompareInfo GetCompareInfo(string name) { return default(System.Globalization.CompareInfo); }
+        // Temproray disable the API's depends on System.Reflection.Assembly till we get it defined in System.Runtime
```

Comment:
please make sure there's an issue assigned to someone for that.

---
In reply to: [76645138](https://github.com/dotnet/corefx/pull/11226#discussion_r76645138) [](ancestors = 76645138)